### DISCUSSION TITLE
iphone  无法联网

### DISCUSSION THREAD

**Discussion by hpsun1109** [getlantern/lantern-forum-cn#902] [General]
已经是专业版了，但是连接之后没有网络，怎么解决，连接状态百度都打不开，别说谷歌这些了，彻底没网了，

**Comment by MaxMKMax** [comment]
@hpsun1109 您好，谢谢您的反馈。蓝灯已经在正视这个问题了。 我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。谢谢您的等待与支持。

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Reply by drockser** [reply]
我也遇到了，我的ticket 号码是77452485

**Reply by hpsun1109** [reply]
> @hpsun1109 您好，谢谢您的反馈。蓝灯已经在正视这个问题了。 我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。谢谢您的等待与支持。
> 
> 您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。
> 
> go to Support (转到支持) select Report an Issue (选择报告问题) choose issue type (选择问题类型) include additional details (包括额外的详细信息) hit Send Report (点击发送报告)
> 
> 当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!
> 
> 有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。
都提交不了，买了一周还上不去网
![image](https://github.com/user-attachments/assets/c2f85d3c-d371-4838-9b53-16be662e242d)

**Comment by hpsun1109** [comment]
@Mike777Khoo 怎么解决呢？

**Comment by coast1017** [comment]
我也遇到同样问题，蓝灯专业版mac可以正常使用联网，也可以成功授权给iphone，但是授权成功后一旦用iphone连接蓝灯，iphone就会失去网络连接，不论是用Wi-Fi还是手机sim卡移动网络都无法联网。所以从电脑端上传了问题（ticket 77534935），不过上传问题时发现无法上传手机截屏图片（一传图片就会回到蓝灯主页）。

**Reply by coast1017** [reply]
![WechatIMG784](https://github.com/user-attachments/assets/a879fa7d-772f-455d-9b45-e681dbcd0a38)
![WechatIMG785](https://github.com/user-attachments/assets/f1baab39-e33c-4e86-8e76-4d0a37ffbbc7)
![WechatIMG785](https://github.com/user-attachments/assets/3c8f5dd5-6f2d-4153-a62d-f42eedc02073)
![WechatIMG786](https://github.com/user-attachments/assets/0b76841a-911a-4d1f-b722-5411f1a992fc)
![WechatIMG786](https://github.com/user-attachments/assets/8356d939-9bda-47c6-b566-477c4c98229c)

**Comment by hpsun1109** [comment]
@Mike777Khoo 一个月也不解决吗？上不去网给我退款吧
